scenarios/kubernetes_e2e.py

```diff
@@ -445,8 +449,7 @@ def create_parser():
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
     parser.add_argument(
-        '--multiple-federations', default=False, action='store_true',
-        help='If we need to run multiple federation control planes in parallel')
+        '--user', default='prow', help='Username used for ssh in local mode')
```

Comment:
This shouldn't be set by the scenario. It should be set by the prow job definition or else the image -- probably the prow job.

If the user runs the scenario they will already have a USER var set correctly